### PR TITLE
#2 - actions added

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #2 
Added pom, actions and other project related files.

### CODE REVIEW

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  pull_request:
+    branches:
+      - master
+jobs:
+  maven-build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest]
+        java: [1.8, 11, 13]
```
Comment (olegmoz) [inline .github/workflows/maven.yml:16]: @olenagerasimova java `1.8` is not compatible with java 11 specified in `pom.xml`, please fix it

File: .github/workflows/pdd.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+name: PDD linter
+"on":
+  push:
+    branches:
+      - master
+  pull_request:
+    brranches:
```
Comment (olegmoz) [inline .github/workflows/pdd.yml:8]: @olenagerasimova there is a typo in `brranches`

File: .github/workflows/xcop.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+name: XCOP linter
+"on":
+  push:
+    branches:
+      - master
+  pull_request:
+    brranches:
```
Comment (olegmoz) [inline .github/workflows/xcop.yml:8]: @olenagerasimova there is a typo in `brranches`

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see comments

**Comment by olenagerasimova** [comment]
@olegmoz thanks, fixed

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, merging - hope action will run after that

**Pull request merged**